File: math/fibonacci/C#/Fibonacci.cs
using System;
using System.Collections.Generic;

namespace Fibonacci
{
    class Fibonacci
    {
        static void Main(string[] args)
        {

            Console.Write("How many Fibonacci numbers do you want?\n");
            int n;
            while (!int.TryParse(Console.ReadLine(), out n))
            {
                Console.Write("Please, insert a number\n");
            }

            foreach (int i in GetFibonacciSequence(n))
            {
                Console.WriteLine(i);
            }

            Console.ReadLine();
        }

        public static List<int> GetFibonacciSequence(int n)
        {

            List<int> fib = new List<int>();
            fib.Add(1);
            fib.Add(1);

            for (int i = 2; i <= n; i++)
            {
                fib.Add(fib[i - 2] + fib[i - 1]);
            }

            return fib;
        }

    }
}

File: math/fibonacci/Lua/fibonacci.lua
--Tested with Lua 5.3.4
io.write("How many Fibonacci numbers do you want? ")
n = io.read("*n")
f1=0
f2=1
for i=1, n do
  temp = f1 + f2
  print(temp)
  f1 = f2
  f2 = temp
end

